src/components/FeatureSelector/AudioFeatureSlider.tsx

```diff
@@ -21,11 +29,39 @@ export default function (props: {
   onFeatureUpdate: (name: string, feature: number[]) => void
 }) {
   const classes = useStylesSlider()
+  const [popupAnchor, setPopupAnchor] = useState(null)
+
   return (
     <div className={classes.root}>
       <Typography id='range-slider' gutterBottom style={{ minWidth: 130 }}>
         {props.feature.name}
       </Typography>
+      <Box style={{ padding: 12 }}>
+        <InfoIcon
+          onMouseEnter={(event: any) => setPopupAnchor(event.currentTarget)}
+          onMouseLeave={() => setPopupAnchor(null)}
+        />
+      </Box>
+      <Popover
```

Comment:
Could the popover go on the text instead of adding an icon.

IMO there is already a lot going on there with the name, slider and delete button